best_practices/templates.rst

```diff
@@ -147,18 +137,11 @@ service in the constructor of the Twig extension:
         }
     }
 
-Lastly define a new service to enable this Twig extension in the app (the service
-name is irrelevant because you never use it in your own code):
-
-.. code-block:: yaml
+And that's it!
 
-    # app/config/services.yml
-    services:
-        app.twig.app_extension:
-            class:     AppBundle\Twig\AppExtension
-            arguments: ['@app.markdown']
-            public:    false
-            tags: [twig.extension]
+Your application will :ref:`autoconfigure <services-autoconfigure>` your twig
+extension and inject a ``Markdown`` instance in it thanks to
+:doc:`autowiring </service_container/autowiring>`.
```

Comment:
After #7906, I have something we can link to that talks about this "default" config. I'm linking to it everywhere pretty consistently. See https://github.com/symfony/symfony-docs/pull/7906/files#diff-37bf298125afc3dde92bb3907795a9e7R291 for an example